alfred-margaret.cabal

```diff
@@ -47,18 +47,20 @@ library
                      , Data.Text.BoyerMoore.Replacer
                      , Data.Text.BoyerMoore.Searcher
                      , Data.Text.Utf16
-  build-depends:       base >= 4.7 && < 5
-                     , containers
-                     , deepseq
-                     , hashable
-                     , primitive
-                     , text
-                     , unordered-containers
-                     , vector
+  build-depends:
+      base             >= 4.7 && < 5
+    , containers       >= 0.6.2 && < 0.7
+    , deepseq          >= 1.4.4 && < 1.5
+    , hashable         >= 1.3.0 && < 1.4
+    , primitive        >= 0.7.0 && < 0.8
+    , text             >= 1.2.4 && < 1.3
+    , unordered-containers  >= 0.2.10 && < 0.3
+    , vector           >= 0.12.1 && < 0.13
   ghc-options:         -Wall -Wincomplete-record-updates -Wincomplete-uni-patterns -O2
   default-language:    Haskell2010
   if flag(aeson) {
-  build-depends:       aeson
+  build-depends:
+      aeson            >= 1.4.7 && < 1.6
```

Comment:
This version bounds are not compatible with the the stage LTS 13.10 of which we say above that we are compatible. I relaxed them a bit more, these work on that older LTS as well:

```
  build-depends:
      base             >= 4.7 && < 5
    , containers       >= 0.6 && < 0.7
    , deepseq          >= 1.4 && < 1.5
    , hashable         >= 1.2.7 && < 1.4
    , primitive        >= 0.6.4 && < 0.8
    , text             >= 1.2.3 && < 1.3
    , unordered-containers  >= 0.2.9 && < 0.3
    , vector           >= 0.12 && < 0.13
  ghc-options:         -Wall -Wincomplete-record-updates -Wincomplete-uni-patterns -O2
  default-language:    Haskell2010
  if flag(aeson) {
  build-depends:
      aeson            >= 1.4.2 && < 1.6
  cpp-options:         -DHAS_AESON
  }
```